.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
```

Comment:
is it voluntary that those are in comments? :thinking:

Reply:
Yes, because it isn't applicable to all PRs so I didn't want to bother PRs where it isn't relevant.

Reply:
It's been nice in other repositories I've worked with that it doesn't show up in your actual post when you create it, but it's there for you to check off all the bullets. 👍